backend/src/main/java/de/bund/digitalservice/ris/adm_vwv/application/converter/transform/FieldsOfLawTransformer.java

```diff
@@ -43,6 +43,7 @@ public List<FieldOfLaw> transform(AkomaNtoso akomaNtoso) {
               false,
               risFieldOfLaw.getValue(),
               risFieldOfLaw.getValue(),
+              null,
```

Comment:
That should be then:
```suggestion
risFieldOfLaw.getNotation(),
```